DTS-STN/Life-Journeys
# Added Table component

# Description

[Content](https://lj-decd.atlassian.net/browse/LJ-91?atlOrigin=eyJpIjoiNWY1NTY3NzMzMjY5NDEyZmE0YjdjMDM0ZjRlYTE5ZTQiLCJwIjoiaiJ9)

Added a 3x2 content table component. Opening the PR to get conversation started on this since I'm unsure if this is the route I should be going with this. Originally tried using react-table instead but ran into some issues when styling. The links don't go anywhere right now by design since we don't have the content for that right now, but can add propTypes later if we keep this format. Tests to follow.

## Test Instructions

1. Pull in branch
2. Type ```npm run dev```
3. Open Accordion to see content table

## Meets Definition of Done
